shopfloor/services/checkout.py

```diff
@@ -478,15 +478,18 @@ def _select_lines_from_package(self, picking, selection_lines, package, **kw):
                     ),
                 },
             )
-        self._select_lines(lines)
+        related_lines = selection_lines.filtered(
+            lambda l: l.package_id != package and not l.shopfloor_checkout_done
+        )
+        lines = self._select_lines(lines, related_lines=related_lines)
```

Comment:
Just for this one, we should not show something out of the currently scanned package

Reply:
Maybe I miss a piece of ctx but I don't understand this change. If you are asking for a specific line that's the only line that you should select, all the others shouldn't be shown on the select package screen IMO :thinking:

Reply:
he dropped this change. When you scan a package, you see only the content of the package. Like it has always been